pkg/asset/manifests/ibmcloud/cloudproviderconfig.go

```diff
@@ -67,6 +74,21 @@ func CloudProviderConfig(infraID string, accountID string, region string, resour
 			G2VPCSubnetNames:         subnetNames,
 		},
 	}
+
+	// Add any IBM Cloud Service Endpoint overrides as necessary
+	for _, endpoint := range serviceEndpoints {
+		switch name := strings.ToLower(endpoint.Name); name {
+		case ibmcloud.IBMCloudServiceIAM:
+			config.Provider.IAMEndpointOverride = endpoint.URL
+		case ibmcloud.IBMCloudServiceVPC:
+			// Trim the version suffix ('/v1', '/v21', etc.) from the VPC URL since the CCM appends it
+			suffixRegex := regexp.MustCompile(`(^.*)/v\d+[/]{0,1}$`)
+			config.Provider.VPCEndpointOverride = suffixRegex.ReplaceAllString(endpoint.URL, "${1}")
+		case ibmcloud.IBMCloudServiceResourceManager:
+			config.Provider.RMEndpointOverride = endpoint.URL
+		}
```

Comment:
would there be any value in logging when this switch falls through?

Reply:
Maybe. Other endpoints the CCM doesn't use (COS, DNS Services, etc.), would fall through, so we'd ignore them in terms of building the CCM config.
But perhaps logging the "ignored" overrides could be helpful if the user has incorrectly configured those required for the CCM, but I'd expect the `validation` steps should filter those issues out
https://github.com/openshift/installer/pull/7632/files#diff-e6306df593d77282680d1864f5e1504e044abdbe681195fbcbeaab0cc3c45cc2R437

I'll add a `default` log call I think though.

Reply:
Added a log call